apps/frontend/src/components/ResponseBox/ResponseBox.tsx

```diff
@@ -98,9 +142,55 @@ export default function ResponseBox({ response, requestOwnerId }: Props) {
         </div>
         <p className={styles.createdDate}>{date}</p>
       </div>
-      <p className={styles.p}>{response.content}</p>
-      <hr className={styles.hr} />
-      <div className={styles.contact}>{contactInfo}</div>
+      {showEditForm ? (
+        <Form noValidate onSubmit={handleSubmit} role="form">
```

Comment:
Would it make sense to extract this as a separate component, or at least in a variable?

Reply:
I think a variable would work best!

Reply:
Hey I just saw this. I’ve moved it in a separate component in my branch so let me know if you’d like to move it back into this one